akka-docs/src/main/paradox/typed/persistence-durable-state.md

```diff
@@ -0,0 +1,391 @@
+  version=AkkaVersion
+  group2=com.typesafe.akka
+  artifact2=akka-persistence-testkit_$scala.binary.version$
+  version2=AkkaVersion
+  scope2=test
+}
+
+You also have to select journal plugin and optionally snapshot store plugin, see 
+@ref:[Persistence Plugins](../persistence-plugins.md).
+
+@@project-info{ projectId="akka-persistence-typed" }
+
+## Introduction
+
+This model of Akka Persistence enables a stateful actor / entity to store the full state afer processing each command instead of using event sourcing. This reduces the conceptual complexity and can be a handy tool for simple use cases. Very much like a CRUD based operation, the API is conceptually simple - a function from current state and incoming command to the next state which replaces the current state in the database. 
+
+```
+(State, Command) => State
+```
+
+The current state is always stored in the database. Akka Persistence would read that state and store it in memory. After processing of the command is finished, the new state will be stored in the database. The processing of the next command will not start until the state has been successfully stored in the database.
+
+The database specific implementations can be added to existing Akka Persistence plugin implementations, starting with the JDBC plugin. The plugin would serialize the state and store as a blob with the persistenceId as the primary key. 
+
+## Example and core API
+
+Let's start with a simple example that models a counter using an Akka persistent actor. The minimum required for a @apidoc[DurableStateBehavior] is:
+
+Scala
+:  @@snip [DurableStatePersistentBehaviorCompileOnly.scala](/akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/DurableStatePersistentBehaviorCompileOnly.scala) { #structure }
+
+Java
+:  @@snip [DurableStatePersistentBehaviorTest.java](/akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/DurableStatePersistentBehaviorTest.java) { #structure }
+
+The first important thing to notice is the `Behavior` of a persistent actor is typed to the type of the `Command`
+because this is the type of message a persistent actor should receive. In Akka this is now enforced by the type system.
+
+The components that make up a `DurableStateBehavior` are:
+
+* `persistenceId` is the stable unique identifier for the persistent actor.
+* `emptyState` defines the `State` when the entity is first created e.g. a Counter would start with 0 as state.
+* `commandHandler` defines how to handle commands and map to appropriate effects e.g. persisting state and replying to actors.
+
+Next we'll discuss each of these in detail.
+
+### PersistenceId
+
+The @apidoc[akka.persistence.typed.PersistenceId] is the stable unique identifier for the persistent actor in the backend
+durabe state store.
+
+@ref:[Cluster Sharding](cluster-sharding.md) is typically used together with `DurableStateBehavior` to ensure
+that there is only one active entity for each `PersistenceId` (`entityId`).
```

Comment:
The `entityId` is not introduced, so it's not clear what it means for a new reader in this sentence?

Reply:
rephrased